operations/firmware/Upload_Olympus_BMC_Recovery_Firmware_into_TFTP_Server.md

```diff
@@ -3,6 +3,11 @@
 `cray-upload-recovery-images` is a utility for uploading the BMC recovery files for ChassisBMCs, NodeBMCs, and RouterBMCs to be served by the *cray-tftp* service. The tool uses the cray cli (*fas*, *artifacts*) and *cray-tftp* to download the s3 recovery images (as remembered by FAS) then upload them into the PVC that is used by *cray-tftp*.
 `cray-upload-recovery-images` should be run on every system.
 
+**Prerequisite:**
+1. CSM software has been installed
```

Comment:
Assuming the order doesn't matter here, these steps can be changed from an ordered list to an unordered list.

Reply:
Updated